### ISSUE TITLE
Provide an option to overwrite the origination endpoint for the CallManager and Dial verb

### ISSUE THREAD

**Issue by psanders** [fonoster/fonoster#300]
**Is your feature request related to a problem? Please describe.**
Provide an option to allow the developers to indicate the origination endpoint (FROM) while using the CallManager or the Dial verb.

**Describe the solution you'd like**

Add an option to allow overwriting the origination endpoint, as long as the service provider permits it.

For the `CallManager,` we will then have something like this:

```
 const Fonoster = require("@fonoster/sdk")
 const callManager = new Fonoster.CallManager()
 
 callManager.call({
   callingAs: "someothernumber"
   from: "9102104343",
   to: "17853178070",
   webhook: "https://https://071e-47-132-137-75.ngrok.io/voiceapp",
 })
   .then(console.log)        // successful response
   .catch(console.error)   // an error occurred
 ```

For the Dial verb, we will have to add the option as follows:

```
await response.dial("17853178070", { callingAs: "someothernumber" });
```

**Describe alternatives you've considered**
None

**Additional context**
None

**Comment by psanders** [comment]
Notes to myself:

By setting to yes the _send_rpid_, _send_pai_, and _rpid_immediate_, I was able to update the From, `P-Asserted-Identity`, and  `Remote-Party-Id` headers on the MediaServer. However, this breaks the flow on Routr since the User part on the From name was updated. Routr displays the following error:

```
[ERROR] javax.sip.SipException: Could not create a message channel for 172.29.0.1:55340/TCP
viaHeader.getReceived() = 172.29.0.1
viaHeader.getRPort() = 37836
[ERROR] Invalid AOR: undefined
```

This error is likely due to the change in the User part of the From header. Also, I noticed I could use the CONNECTEDLINE(name) and CONNECTEDLINE(num) variables to update the Display name and User part of the From header, respectively.

> CALLERID did not have any effect on updating the From value

It looks like we will need to update Routr to bypass the header `P-Asserted-Identity`, and  `Remote-Party-Id` if the MediaServer sets them.
